tox.ini

```diff
@@ -18,7 +18,7 @@ deps =
   igittigitt
   # TODO: move to a released version
   git+https://github.com/pmeier/pytorch_testing_utils
-commands = pytest -c pytest.ini --durations=25
+commands = pytest -Werror -c pytest.ini --durations=25
```

Comment:
Lets put that into `pytest.ini`, since this should apply to all runs not just the integration tests.

Reply:
Thanks for the suggestion. This should be fixed now in the recent commit.